cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,80 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import HomePageObject from "../support/pages/home.pageObject";
+import SettingsPageObject from "../support/pages/settings.pageObject";
+
+const settingsPage = new SettingsPageObject();
+const homePage = new HomePageObject();
+
 describe('Settings page', () => {
+  let user;
+  let updateUser;
+
   before(() => {
 
   });
 
   beforeEach(() => {
-
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+      cy.register(user.email, user.password, user.username);
+    });
+    cy.task('generateUpdateUser').then((generateUpdateUser) => {
+      updateUser = generateUpdateUser;
+    });
   });
 
   it('should provide an ability to update username', () => {
+    settingsPage.visit();
+    settingsPage.typeUsername(updateUser.username);
 
+    settingsPage.clickOnUpdateSettingsBtn();
+
+    settingsPage.visit();
+
+    settingsPage.verifyUsernameUpdate(updateUser.username);
   });
 
   it('should provide an ability to update bio', () => {
+    settingsPage.visit();
+    settingsPage.typeBio(updateUser.bio);
+
+    settingsPage.clickOnUpdateSettingsBtn();
+
+    settingsPage.visit();
 
+    settingsPage.verifyBiolUpdate(updateUser.bio);
```

Comment:
There is a typo in the method name `verifyBiolUpdate`. It should be corrected to `verifyBioUpdate` to match the intended functionality.